docs/content/TreeView.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: TreeView
+componentId: tree_view
+status: Draft
+description: A hierarchical list of items where nested items can be expanded and collapsed.
+---
+
+## Examples
```

Comment:
Love these examples, noticed a weird styling issue when viewing the deployment:

https://user-images.githubusercontent.com/3901764/189394668-81827919-916f-4a06-9926-bc3e379fdb16.mov

Didn't see this at all in storybook, though 🤔

Reply:
Fixed. Good catch!